### PR TITLE
Change list-returning endpoints to plural

### LINKED ISSUE #96: Organizations endpoint should be plural
Collections should be in plural from. https://github.com/RedHatInsights/insights-results-aggregator/blob/master/openapi.yml#L9 should be changed to `/organizations:`

### PR DESCRIPTION
**Pull request by natiiix**
# Description

Modified endpoint names to match the requirements.

The tests all seem to pass, but I assume this change will break something.

Fixes #96 

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## Testing steps

❓

### CODE REVIEW

**Comment by natiiix** [comment]
@tisnik Oops, didn't notice that.

File: server/server.go [hunk 1]
```diff
@@ -18,9 +18,9 @@ limitations under the License.
 // Insights results aggregator service. In current version, the following
 // REST API endpoints are available:
 //
-// API_PREFIX/organization - list of all organizations (HTTP GET)
+// API_PREFIX/organizations - list of all organizations (HTTP GET)
 //
-// API_PREFIX/cluster/{organization} - list of all clusters for given organizations (HTTP GET)
```
Comment (quarckster) [inline server/server.go]: > list of all clusters for given organizations (HTTP GET)
Does it mean that I can provide more than one orgID separated by a comma and receive a list of clusters?

Comment (tisnik) [reply server/server.go]: nope, should be organizations/organization

**Comment by natiiix** [comment]
> Basically looks good, but I think we need to update `report` request as well, to be a part of `organizations/{organization}/clusters` chain

@bond95 Not sure what you mean, but we cannot do `/organizations/{orgId}/cluters/{clusterId}/report`.

**Pull request merged**